src/main/java/com/mercadopago/client/preapproval/PreApprovalAutoRecurringCreateRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.mercadopago.client.preapproval;
+
+import java.math.BigDecimal;
+import lombok.Builder;
+import lombok.Getter;
+
+/** Recurring data. */
+@Getter
+@Builder
+public class PreApprovalAutoRecurringCreateRequest {
+  /** Currency ID. */
+  private final String currencyId;
+
+  /** Recurring amount. */
+  private final BigDecimal transactionAmount;
+
+  /** Recurring frequency. */
+  private final Integer frequency;
+
+  /** Recurring frequency type (days or months). */
+  private final String frequencyType;
+
+  /** Recurring start date. Must follow the patter ("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") */
+  private final String startDate;
```

Comment:
```suggestion
  /** Recurring start date. Must follow the pattern ("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") */
  private final String startDate;
```
Seria melhor utilizar o tipo Date e o Serializer se encarregar de garantir o formato ao interagir com a API?

Reply:
O ideal é usar o framework `java.time` que foi introduzido no java 8.
Nele tem novas classe como `LocalDateTime` e `OffsetDateTime`. Para o nosso caso acredito que `OffsetDateTime` é melhor.
Dá uma olhada nesse [tutorial](https://docs.oracle.com/javase/tutorial/datetime/TOC.html) da Oracle.
Obs: Talvez tenha que configurar o Gson para funcionar corretamente com esses tipos.

Reply:
Por enquanto mantive o tipo Date para todas as datas e fiz um ajuste no serializer para que o formato parseado seja compatível com a api de preapproval e as demais. Ainda nessa sprint vamos debater sobre esse tema e definir como vamos ser em relação ao tratamento de datas em outro PR.